frontend/src/app/edit-claim/edit-claim.component.html

```diff
@@ -77,12 +77,16 @@ <h1 [lyTyp]="'display2'" class="mbs">{{ claim.vehicle.vin }}</h1>
             <ly-grid item col="6">
               <p>{{ file.name }}</p>
             </ly-grid>
-            <!-- TODO: Fix unauthorized delete error -->
-            <!-- <ly-grid item col="3">
-              <button ly-button raised bg="warn" (click)="deleteFile(file.name)">Delete</button>
-            </ly-grid> -->
-            <ly-grid item col="3">
-              <button ly-button raised bg="primary" (click)="downloadFile(file.hash, file.name)">Download</button>
+            <ly-grid item col="3" *ngIf="file.status === 0; else restoreTemplate">
+              <button ly-button raised bg="warn" (click)="archiveFile(file.name)" class="width-100">Archive</button>
+            </ly-grid>
+            <ng-template #restoreTemplate>
+              <ly-grid item col="6">
+                <button ly-button raised outlined color="primary" (click)="restoreFile(file.name)" class="width-100">Restore</button>
```

Comment:
Likewise here. If we know the file is archived, we shouldn't openly let users see it until they are editing the claim with the Edit Vehicle button